### PR TITLE
1908-uninstall-guide-linux

### PR DESCRIPTION
**Pull request by MathiasMagnus**
Closes: #1908

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
 
 Welcome to AMD ROCm's documentation!
 
+## What is ROCm?
+
+ROCm is a brand name for the ROCm open software platform (for software) or the
```
Comment (saadrahim) [inline docs/index.md:7]: Remove please

File: docs/quick_start.md [hunk 2]
```diff
@@ -1,292 +1,360 @@
...
+
+## Reboot the system
+
+Loading the new driver requires a reboot of the system.
+
+```shell
+sudo reboot
+```
+
+## Uninstallation
```
[the remark this answers is not in the document]
Comment (MathiasMagnus) [reply docs/quick_start.md]: Addressed

File: docs/index.md [hunk 3]
```diff
@@ -2,6 +2,19 @@
...
+ROCm open platform ecosystem (includes hardware like FPGAs or other CPU
+architectures).
+
+```{note}
+ROCm no longer functions as an acronym.
```
[the remark this answers is not in the document]
Comment (MathiasMagnus) [reply docs/index.md]: Addressed

File: docs/index.md (hunk 1 above)
Comment (MathiasMagnus) [reply docs/index.md]: Addressed

File: docs/index.md
[the remark this answers is not in the document]
Comment (MathiasMagnus) [reply docs/index.md:1]: Reverted

**Review by saadrahim — APPROVED** [review]
LGTM

**Pull request merged**